main/src/main/java/cgeo/geocaching/log/LogCacheActivity.java

```diff
@@ -357,7 +358,11 @@ private void initializeFavoriteCheck() {
             final int favoritePoints = ((ILoggingWithFavorites) loggingManager).getFavoritePoints();
             if (favoritePoints > 0) {
                 binding.favoriteCheck.setVisibility(View.VISIBLE);
-                binding.favoriteCheck.setText(res.getQuantityString(R.plurals.fav_points_remaining, favoritePoints, favoritePoints));
+                if (connector instanceof OCApiConnector) {
```

Comment:
Please don't put any  logic here which depends on the used connector. The whole idea of the connector concept is to keep the differences of the connected platforms out of the common activities. 

If I understand correctly you want to differentiate the wording "favorite" (gc) from "recommendation" (oc). If you really feel the need to do so please abstract it out towards the IFavoriteCapability interface (e.g. add a method which gives the correct resId for the word to use in display).  

Personally I feel we should have the same wording in c:geo across platforms/connectors , means we should use "Favorite" for all platforms, but that's debatable of course (hence the suggestion to abstract it out to IFavoriteCapability interface). What is imho not debatable however is that we use the same wording in code everywhere - and here we use "Favorite" (eg. IFavoriteCapability and other classes) and not "recommendation".

Reply:
I prepared a new method in ILoggingWithFavorites interface (to be consistent with already implemented ``` ((ILoggingWithFavorites) loggingManager).getFavoritePoints();```.

From technical point of view I fully understand your concerns about unified wording. For a geocacher who caching only opencaching.pl I wonder 'favorite' can be a bit missleading, especially for fresh players.
I prepared a change where I added oc-only wording but with suggestion of @Lineflyer. Of course if cgeo lead programmers have strong option about this kind of oddities I can change my PR to use 'favorite' text also for OC.